chapter6_smartview/chapter6_smartview.md

```diff
@@ -0,0 +1,197 @@
+# Chapter 6 SmartView
+SmartView is an umbrella term for one of the live streaming technologies offered by Sitelink3D v2. Compared with MFK Live (Machine Forward Kinematics), SmartView involves less programming investment to start using and is capable of providing information at various resolutions and with configurable server side filtering. These features facilitate a set of live streaming capabilities that can be used to implement expressions that make sense to the end user.
+
+## SmartView Microservice
+The ```smart_view``` microservice manages requests for SmartView data. It streams data using SSE (server sent events) and manages the lifetimes of SmartApps discussed below.
+
+## SmartApps
+SmartView delivers functionality by means of SmartApps. A SmartApp is simply a piece of serverside JavaScript that provides information about a specific bounded context of site operations that a user may wish to subscribe to. Each SmartApp has a particular purpose and is only concerned with providing information within its domain. SmartApp examples include:
+
+- Machine locations and state.
+- Weight payloads sent between machines.
+- Haul activity.
+
+A SmartApp is identified by three settings: 
+
+- Key which is a string like ‘topcon/machines/machine_list’.
+- Version which is a UUID. 
+- Start time.
+
+To subscribe to a SmartApp, you will need to know the following settings:
+1. Site of interest: you need the site identifier (hereafter called SITE).
+2. SmartApp of interest (hereafter called APP_B64): SmartApps have two distinguishing parts of their identity as discussed above:
```

Comment:
Suggest delete the "hereafter" here. APP_B64 is constructed according to a recipe below.

Reply:
Done